src/main/java/io/confluent/connect/jdbc/util/DateTimeUtils.java

```diff
@@ -20,56 +20,52 @@
 import java.util.Calendar;
 import java.util.Date;
 import java.util.GregorianCalendar;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.TimeZone;
 
 public class DateTimeUtils {
 
-  public static final TimeZone UTC = TimeZone.getTimeZone("UTC");
+  private static final ThreadLocal<Map<TimeZone, Calendar>> TIMEZONE_CALENDARS =
+      ThreadLocal.withInitial(HashMap::new);
```

Comment:
HashMap's `computeIfAbsent()` method used below is not threadsafe, and although currently the methods in this utility are only called from the task's thread, it might be worth using `ConcurrentHashMap::new` here so that this class is able to be called concurrently from multiple threads.

Reply:
In a first iteration i was thinking to do as you suggested, but then I thought that this is not necessary anymore because the map is being used as a threadlocal variable . In the way the DateTimeUtils class is currently built (based on ThreadLocal variables) there's  no way of falling into concurrency issues.

Take into account that if the class wouldn't be based on ThreadLocal holders, the java.util.Calendar obtained from the method io.confluent.connect.jdbc.util.DateTimeUtils#getTimeZoneCalendar could be used sinmultaneously   in multiple threads, which would cause problems (because java.util.Calendar is not thread-safe).

Reply:
Ah, that's a great point. The calendars are not threadsafe, so the thread local approach doesn't need to be concurrent.